### PR TITLE
Leeroy auth

### PR DESCRIPTION
**Pull request by leeroyp**

### CODE REVIEW

**Review by dghazi12 — APPROVED** [review]
Things Checked : 

- Existing email and password works
- Tried to sign in with no email address, alerts to input email 👍 
- Tried to sign in without an @ for email address, wouldn't allow me to sign up so that works good as well
- Tried to sign in without a password input, wouldn't allow me 👍 
- Tried the forget my password, able to update password and follow link received in my email address 👍 
- Able to sign up as new user, tried same things as above and works the same way 👍 
- Also, if passwords don't match, it will not allow a sign up 👍 
- Basically, everything works good 👍
